BaragonCore/src/main/java/com/hubspot/baragon/models/BaragonRequest.java

```diff
@@ -64,19 +68,20 @@ public BaragonRequest(@JsonProperty("loadBalancerRequestId") String loadBalancer
     this.replaceUpstreams = Objects.firstNonNull(replaceUpstreams, Collections.<UpstreamInfo>emptyList());
     this.noValidate = Objects.firstNonNull(noValidate, false);
     this.noReload = noReload;
+    this.noServiceUpdate = noServiceUpdate;
```

Comment:
Yeah, that'd be a bit of a mess. Should we validate such that the two options are not allowed at the same time?